Winapp2.ini

```diff
@@ -3758,20 +3621,20 @@ Default=False
 FileKey1=%LocalAppData%\VirtualStore\Program Files*\BOINC|stdout*.*;stderr*.*
 FileKey2=%ProgramFiles%\BOINC|stdout*.*;stderr*.*
 
-[Borderlands*]
-Section=Games
-Detect1=HKCU\Software\Valve\Steam\Apps\8980
-Detect2=HKLM\Software\Borderlands
-Default=False
-FileKey1=%Documents%\My Games\Borderlands*\WillowGame\Logs|*.log
-
 [Borderlands 2*]
 Section=Games
 Detect=HKCU\Software\Valve\Steam\Apps\49520
 DetectFile=%Documents%\My Games\Borderlands 2
 Default=False
 FileKey1=%Documents%\My Games\Borderlands 2\WillowGame\Logs|*.*
 
+[Borderlands*]
```

Comment:
Maybe we could combine Borderlands 1 and 2 into 1 entry and call it Borderlands 1/2*?